iocage/tests/unit_tests/1001_lib_start_test.py

```diff
@@ -0,0 +1,74 @@
+import unittest.mock as mock
+import pytest
+from iocage.lib.ioc_start import find_bridge_mtu
+
+@mock.patch('iocage.lib.ioc_start.checkoutput')
+def test_should_return_mtu_of_first_member(mock_checkoutput):
+    mock_checkoutput.side_effect = [bridge_if_config, member_if_config]
+
+    mtu = find_bridge_mtu('bridge0')
+    assert mtu == '1500'
+    mock_checkoutput.assert_has_calls([mock.call(["ifconfig", "bridge0"]),
+                                       mock.call(["ifconfig", "bge0"])])
+
+@mock.patch('iocage.lib.ioc_start.checkoutput')
+def test_should_return_mtu_of_first_member_with_description(mock_checkoutput):
+    mock_checkoutput.side_effect = [bridge_with_description_if_config,
+                                    member_if_config]
+
+    mtu = find_bridge_mtu('bridge0')
+    assert mtu == '1500'
+    mock_checkoutput.assert_has_calls([mock.call(["ifconfig", "bridge0"]),
+                                       mock.call(["ifconfig", "bge0"])])
+
+@mock.patch('iocage.lib.ioc_start.checkoutput')
+def test_should_return_default_mtu_if_no_members(mock_checkoutput):
+    mock_checkoutput.side_effect = [bridge_with_no_members_if_config,
+                                    member_if_config]
+
+    mtu = find_bridge_mtu('bridge0')
+    assert mtu == '1500'
+    mock_checkoutput.called_with(["ifconfig", "bridge0"])
+
+bridge_if_config = """bridge0: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> metric 0 mtu 1500
+        ether 00:00:00:00:00:00
+        nd6 options=1<PERFORMNUD>
+        groups: bridge
+        id 00:00:00:00:00:00 priority 32768 hellotime 2 fwddelay 15
+        maxage 20 holdcnt 6 proto rstp maxaddr 2000 timeout 1200
+        root id 00:00:00:00:00:00 priority 32768 ifcost 0 port 0
+            member: bge0 flags=143<LEARNING,DISCOVER,AUTOEDGE,AUTOPTP>
+            ifmaxaddr 0 port 1 priority 128 path cost 20000
+"""
+
+bridge_with_description_if_config = """bridge0: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> metric 0 mtu 1500
```

Comment:
I'm aware these lines are >80 columns. I don't think it adds to the readability to artificially break them.